typescript/packages/plugins/balmy/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "@goat-sdk/plugin-balmy",
```

Comment:
Better to copy the structure from another plugin. It's missing some fields and the way it declares the dependencies.

You can also generate the boilerplate of a plugin using a script now :) check it out here: https://ohmygoat.dev/plugins#using-the-plugin-generator

Reply:
yes

Reply:
This was not resolved

Reply:
copied the structure from another plugin, is something missing ?